dev-java/core-specs-alpha/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
Add yourself and proxy maint

Reply:
Done https://github.com/tgbugs/gentoo/blob/039a84b15cf374012f07f6f4661048763ae202ac/dev-java/core-specs-alpha/metadata.xml